src/dgraph.ts

```diff
@@ -1,11 +1,16 @@
 import fetch from 'node-fetch';
 import { GraphQLResponse, AuthHeaderField } from '@slash-graphql/lambda-types';
 
-export async function graphql(query: string, variables: Record<string, any> = {}, authHeader?: AuthHeaderField): Promise<GraphQLResponse> {
+export async function graphql(query: string, variables: Record<string, any> = {}, authHeader?: AuthHeaderField, accessJWT?:string): Promise<GraphQLResponse> {
```

Comment:
Can we make this class just like dql too?

Reply:
don't get it. Could you suggest the change?

Reply:
No. graphql is a function passed to the execution context.
dql is a class passed to the context and user can invoke dql.query and dql.mutate
Don't want to change the way those objects are used for backward compatibility. 
So for the moment we keep graphql and dql logic separate.